endpoints.md

```diff
@@ -43,9 +42,8 @@
 * [/api/workflow/pooltasks](pooltasks.md)
 * [/api/workflow/claimedtasks](claimedtasks.md)
 * [/api/tools/feedbacks](feedbacks.md)
-
-## Endpoints Under Development/Discussion
 * [/api/authz/resourcepolicies](resourcepolicies.md)
+* [/api/config/bulkaccessconitionoptions](bulkaccessconditionoptions.md)
```

Comment:
Misspelling in the path... you have "conition" when it should be "condition"

Reply:
done